script/lookup-commit.sh

```diff
@@ -237,15 +237,6 @@ test notes != "$mode" || {
 			continue
 		fi
 
-		amlog="$(git -C "$GITGIT_DIR" notes --ref=refs/notes/gitster-amlog show "$commit" 2>/dev/null | head -n 1)"
```

Comment:
For the record: `gitster-amlog` was not fetched automatically by any code in GitGitGadget or its Azure Pipelines for quite a while, therefore this code was functionally dead anyway.